pkgs/development/embedded/edl/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, fetchFromGitHub, python3Packages }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "edl";
+  version = "unstable-2022-09-07";
+
+  src = fetchFromGitHub rec {
+    owner = "bkerler";
+    repo = "edl";
+    rev = "f6b94da5faa003b48d24a5f4a8f0b8495626fd5b";
+    fetchSubmodules = true;
+    hash = "sha256-bxnRy+inWNArE2gUA/qDPy7NKvqBm43sbxdIaTc9N28=";
```

Comment:
Probably some case sensitive file

```
Submodule 'Loaders' (https://github.com/bkerler/Loaders) registered for path 'Loaders'
Initialized empty Git repository in /nix/store/n86pb628wyxnbk50hpmx0qk9jms3fcxy-source/Loaders/.git/
remote: Enumerating objects: 787, done.
remote: Counting objects: 100% (787/787), done.
remote: Compressing objects: 100% (256/256), done.
remote: Total 787 (delta 554), reused 729 (delta 530), pack-reused 0
Receiving objects: 100% (787/787), 47.45 MiB | 2.99 MiB/s, done.
Resolving deltas: 100% (554/554), done.
From https://github.com/bkerler/Loaders
 * branch            HEAD       -> FETCH_HEAD
Updating files: 100% (714/714), done.
Switched to a new branch 'fetchgit'
removing `.git'...
error: hash mismatch in fixed-output derivation '/nix/store/g9yx7lxlb2j0jgiax41av615s4rv9fj9-source.drv':
         specified: sha256-bxnRy+inWNArE2gUA/qDPy7NKvqBm43sbxdIaTc9N28=
            got:    sha256-yPQikrML6MHf2l3vrCt7OMQCwkOtRX9MfeULUQMUVGs=
```

If you don't know how to fix that, just mark it broken for darwin

Reply:
Marked broken on Darwin and left a comment